package.json

```diff
@@ -304,7 +304,7 @@
     "webpack-cli": "^4.7.2"
   },
   "dependencies": {
-    "@influxdata/flux-lsp-node": "^0.6.2",
+    "@influxdata/flux-lsp-node": "^0.6.4",
```

Comment:
was this meant to skip a version?

Reply:
Yeah, we never released a vsflux that used `0.6.3`.